test/test_image.py

```diff
@@ -1,39 +1,44 @@
-import os
-import io
 import glob
+import io
+import os
 import unittest
-import sys
 
+import numpy as np
 import torch
-import torchvision
 from PIL import Image
+
 from torchvision.io.image import (
     decode_png, decode_jpeg, encode_jpeg, write_jpeg, decode_image, read_file,
     encode_png, write_png, write_file)
-import numpy as np
-
-from common_utils import get_tmp_dir
-
 
 IMAGE_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets")
 FAKEDATA_DIR = os.path.join(IMAGE_ROOT, "fakedata")
 IMAGE_DIR = os.path.join(FAKEDATA_DIR, "imagefolder")
 DAMAGED_JPEG = os.path.join(IMAGE_ROOT, 'damaged_jpeg')
+TMP_DIR = os.path.join(IMAGE_ROOT, 'tmp_folder')
```

Comment:
Have you tried using the `get_tmp_dir()` that exists in `common_utils`? If possible we should avoid writing directly in the `IMAGE_ROOT` folder because killing the test is likely to leave temporary files within the project.

Reply:
I opted for `setUpClass()` and `tearDownClass()` directly since in that case we do not repeat ourselves by using `get_tmp_dir()` in tests. Also I think it better reflects a real usage for some tests, for example, `read_file()` function tested in `test_read_file()` expects a path to an image so I would rather test it that way then rely on a contextmanager that `get_tmp_dir()` is (although it could be a exactly the same at the end of the day - I am not sure of the low level implementations). Also we could handle killing of the tests (and therefore leaving temporary files) by using try, except and finally statement in `setUpClass()` just as in `get_tmp_dir()`. However, if your preference for `get_tmp_dir()` is still strong I can rewrite it. Let me know.

Reply:
Thanks @kondela. I don't think there is any repetition using `get_tmp_dir()`. You just call the function once when needed to get a temporary folder. See the original usage on test `test_read_file()` for more info.

As per the original issue, I think it's worth moving all but necessary writes out of the asset folder.

Reply:
I agree with @datumbox , it would be great if we could use `get_tmp_dir` instead.